### PR TITLE
Add AcceleratorStats to cri_stats_provider for 1.19

### PR DESCRIPTION
**Pull request by ruiwen-zhao**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
This is a cherry-pick of https://github.com/kubernetes/kubernetes/pull/96873
**Which issue(s) this PR fixes**:
See https://github.com/kubernetes/kubernetes/pull/96873

Fixes #

**Special notes for your reviewer**:
See https://github.com/kubernetes/kubernetes/pull/96873

**Does this PR introduce a user-facing change?**:

```release-note
AcceleratorStats will be available in the Summary API of kubelet when cri_stats_provider is used.
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/kubelet/stats/cri_stats_provider.go [hunk 1]
```diff
@@ -785,6 +785,25 @@ func (p *criStatsProvider) addCadvisorContainerStats(
...
+		cs.CPU = cpu
+	}
+	if memory != nil {
+		cs.Memory = memory
+	}
+	accelerators := cadvisorInfoToAcceleratorStats(caPodStats)
+	cs.Accelerators = accelerators
+}
+
+func (p *criStatsProvider) addCadvisorContainerCPUAndMemoryStats(
```
Comment (SergeyKanzhelev) [inline pkg/kubelet/stats/cri_stats_provider.go:799]: so the test for this functionality mocks cAdvisor results. Can you confirm that cAdvisor that was used in 1.19 was producing these metrics? Just a review of commits diff may be enough I think

Comment (ruiwen-zhao) [reply pkg/kubelet/stats/cri_stats_provider.go:799]: We have actually tested this end-to-endly, and have verified that accelerator metrics are available thru summary API

**Review by hasheddan — APPROVED** [review]
/lgtm

**Pull request merged**